### ISSUE TITLE
Problem with png file detection

### ISSUE THREAD

**Issue by tknopp** [JuliaIO/FileIO.jl#210]
Hi I get this
```
  FileIO.File{FileIO.DataFormat{:UNKNOWN}}("/Users/knopp/.julia/dev/MRIReco/test/recogre.png") couldn't be recognized by FileIO.
  
  Stacktrace:
   [1] #save#30(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::FileIO.File{FileIO.DataFormat{:UNKNOWN}}, ::Array{Gray{Float64},2}) at /Users/knopp/.julia/packages/FileIO/lt9I0/src/loadsave.jl:199
   [2] save(::FileIO.File{FileIO.DataFormat{:UNKNOWN}}, ::Array{Gray{Float64},2}) at /Users/knopp/.julia/packages/FileIO/lt9I0/src/loadsave.jl:199
   [3] #save#15(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::String, ::Array{Gray{Float64},2}) at /Users/knopp/.julia/packages/FileIO/lt9I0/src/loadsave.jl:118
   [4] save(::String, ::Array{Gray{Float64},2}) at /Users/knopp/.julia/packages/FileIO/lt9I0/src/loadsave.jl:118
   [5] macro expansion at /Users/knopp/.julia/dev/MRIReco/test/testISMRMRD.jl:35 [inlined]
   [6] macro expansion at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/Test/src/Test.jl:1083 [inlined]
   [7] top-level scope at /Users/knopp/.julia/dev/MRIReco/test/testISMRMRD.jl:5
   [8] include at ./boot.jl:317 [inlined]
   [9] include_relative(::Module, ::String) at ./loading.jl:1038
   [10] include(::Module, ::String) at ./sysimg.jl:29
   [11] include(::String) at ./client.jl:388
   [12] top-level scope at none:0
   [13] include at ./boot.jl:317 [inlined]
   [14] include_relative(::Module, ::String) at ./loading.jl:1038
   [15] include(::Module, ::String) at ./sysimg.jl:29
   [16] include(::String) at ./client.jl:388
   [17] top-level scope at none:0
   [18] eval(::Module, ::Any) at ./boot.jl:319
   [19] macro expansion at ./logging.jl:317 [inlined]
   [20] exec_options(::Base.JLOptions) at ./client.jl:219
   [21] _start() at ./client.jl:421
```

I am not sure what is going on, but it happens when I uncomment this line:
https://github.com/MagneticResonanceImaging/MRIReco.jl/blob/master/test/testISMRMRD.jl#L35

**Comment by tknopp** [comment]
@SimonDanisch

**Comment by SimonDanisch** [comment]
Probably a corrupted file already exists?

**Comment by MarkusQ** [comment]
This looks like a duplicate of #132

**Comment by tknopp** [comment]
yep, even better would be a fix for #132. ;-)
